### ISSUE TITLE
Menu bar was the only part that appeared today

### ISSUE THREAD

**Issue by katstew** [al-caughey/Google-Meet-Attendance#106]
I have been using Google Meet Attendance for months now without issues.  Thank you for your hard work.  Today I launched my class Meet and the menu bar appeared (by itself without any space for names underneath - image attached).  None of the functionality was available.    The checkmark beside the microphone,  leave call and camera icons wasn't there.  The Chrome extension is still there.  I tried restarting to no avail.  Three hours later, it was back to working.  Thoughts?
![Meet issue](https://user-images.githubusercontent.com/78039049/105877238-c13f0800-5fcd-11eb-8642-1cc8ef955b21.PNG)

**Comment by AlCaughey** [comment]
that's not supposed to happen.  That would typically indicate that there
was a Javascript error but not sure why given that it appears to be working
again now (right?)

**Comment by katstew** [comment]
yes.  Interestingly, it didn't do an automatic download at the end of the class, but it sent me a clean copy :-)


Again, thanks for your hard work.  This program has been so very useful for tracking attendance.

?Kathryn Stewart
Math Department Head
Newmarket HS
YRDSB
905-895-5159 ext. 424
The point of technology..."isn't to help schools do the same old things better, but to help teachers and students do better things" -- Chris Dede, Harvard University
________________________________
